02Lesson/02Project/index.html

```diff
@@ -1,10 +1,53 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
-</head>
-<body>
-  <!-- You code start here -->
-</body>
+  <head>
+    <title>
+      The Daily Planet
+    </title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css" type="text/css"/>
+  </head>
+  <body>
+    <h1>
+      Daily Planet!!
+    </h1>
+    <nav>
+      <a href="http://www.espn.com/">Sports</a> |
+      <a href="http://www.statesman.com/news/local/">Local</a> |
+      <a href="http://www.wsj.com/news/economy">Economy</a>
+    </nav>
+    <main>
+      <section>
+        <article>
+          <h2>
+            What's Going On In the World
+          </h2>
+          <h3>
+            Orangutans: Offended to be compared to Donald Trump
+          </h3>
+          <img src="https://i.ytimg.com/vi/GeWc-hKxWu0/hqdefault.jpg " alt="Picture of Donald Trump yelling " height="360" width="480">
+          <h3>
+            Afraid Donald May Win? So is He!
+          </h3>
+          <img src="https://malialitman.files.wordpress.com/2015/09/donald-trump-hair-four.jpg " alt="Profile of sad Organgutan " height="360" width="400" >
+          <h3>
+            Climate Change Is Still Real.
+          </h3>
+          <img src="http://barentsobserver.com/sites/barentsobserver.com/files/main/articles/polar_bear_jumping_edited.jpg " alt="Picture of Polar Bear jumping between broken ice " height="360" width="400" >
+        </article>
+      </section>
+      <aside>
+        <h5>
+          Brought to you by
+        </h5>
+        <img src="http://vignette4.wikia.nocookie.net/logopedia/images/e/eb/Shell_1948.png/revision/latest?cb=20120831065500" alt="vintage Shell logo " >
+        <h5>
+          -and-

```

Comment:
That word does not really deserve to be a headline (h5)... :-) Now that we know CSS, a `span` with some CSS styling would be more appropriate...